mmdet/models/roi_heads/bbox_heads/convfc_bbox_head.py

```diff
@@ -132,8 +132,8 @@ def init_weights(self):
                     nn.init.xavier_uniform_(m.weight)
                     nn.init.constant_(m.bias, 0)
 
-    def forward(self, x):
```

Comment:
After discussing with @hellock and @ZwwWayne , we think we may keep `convfc_bbox_head.py` unchanged. 
Then, we could add a new box head for this function.